### PR TITLE
fix: refine validation rules as part of pre-1.0 work (group 1)

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
This PR contains changes to the first group of validation rules to be refined for the 1.0 release.

### General checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Dependencies have been updated as needed
- [ ] .secrets.baseline updated as needed?

#### Checklist for adding a new validation rule:
- [ ] Added new validation rule definition (packages/ruleset/src/rules/*.js, index.js)
- [ ] If necessary, added new validation rule implementation (packages/ruleset/src/functions/*.js, updated index.js)
- [ ] Added new rule to default configuration (packages/ruleset/src/ibm-oas.js)
- [ ] Added tests for new rule (packages/ruleset/test/*.test.js)
- [ ] Added docs for new rule (docs/ibm-cloud-rules.md)

### CODE REVIEW

File: .cra/.cveignore [hunk 1]
```diff
@@ -1,6 +1 @@
-[
```
Comment (padamstx) [inline .cra/.cveignore:1]: don't need this exception any more

File: packages/ruleset/src/functions/array-attributes.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+const isPlainObject = require('lodash/isPlainObject');
```
Comment (padamstx) [inline packages/ruleset/src/functions/array-attributes.js:1]: This used to be array-boundary.js

File: packages/ruleset/src/functions/property-attributes.js [hunk 3]
```diff
@@ -53,36 +64,10 @@ function checkPropertyAttributes(schema, path) {
     }
   }
 
-  if (isArraySchema(schema)) {
```
Comment (padamstx) [inline packages/ruleset/src/functions/property-attributes.js:56]: This was moved to ibm-array-attributes.

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

**Review by pyrooka — APPROVED** [review]
LGTM!

**Pull request merged**